calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg

```diff
@@ -0,0 +1,5 @@
+# Generated by confd
```

Comment:
It's possible that I do not fully understand the details here, but I think there are now 2 reasons why we should not need to have `calico_node/filesystem/etc/calico/confd/config/custom_filters.cfg` and `calico_node/filesystem/etc/calico/confd/config/custom_filters6.cfg` in this repo.

1. We have merged https://github.com/projectcalico/libcalico/pull/150 - which means that the etcd paths `/calico/bgp/v1/global/custom_filters/v4` and `/calico/bgp/v1/global/custom_filters/v6` _will_ exist.

2. If (1) was not true, it should still be possible in principle to modify `calico_node/filesystem/etc/calico/confd/conf.d/custom_filters.toml` and `calico_node/filesystem/etc/calico/confd/templates/custom_filters.cfg.template` (and the 6 versions) to use `exists` for the `custom_filters` dir.  (I guess you would need to change `prefix` from `/calico/bgp/v1/global/custom_filters` to `/calico/bgp/v1/global/`.)

With either (1) or (2), execution of `calico_node/filesystem/etc/calico/confd/conf.d/custom_filters.toml` should proceed without error and generate `/etc/calico/confd/config/custom_filters.cfg`, so it should not be necessary to have a copy of `/etc/calico/confd/config/custom_filters.cfg` in this repo.

Am I missing something?